src/app/games/coerceo/coerceo.component.ts

```diff
@@ -48,21 +48,20 @@ export class CoerceoComponent extends TriangularGameComponent<CoerceoRules,
         this.encoder = CoerceoMove.encoder;
         this.tutorial = new CoerceoTutorial().tutorial;
         this.SPACE_SIZE = 70;
-        this.showScore = true;
         this.updateBoard();
     }
     public updateBoard(): void {
         this.chosenCoord = MGPOptional.empty();
         this.state = this.rules.node.gameState;
-        this.scores = this.state.captures;
+        this.scores = MGPOptional.of(this.state.captures);
         this.tiles = this.state.tiles;
-        const move: CoerceoMove = this.rules.node.move;
-        if (move == null) {
+        const move: MGPOptional<CoerceoMove> = this.rules.node.move;
+        if (move.isAbsent()) {
```

Comment:
could you do me a favor and replace by
if (move.isPresent()) {
....this.showLastMove();
else {
....this.hideLastMove();
}

Not making you already future word, just "prefer if true else" :D !
Also, how typomaniac am I, right :D ?

Reply:
le soucis c'est qu'un cous "move == null" c'est le true, un coup "move != null" c'est le true :D (car != null = isPresent)